spec/spec_helper.rb

```diff
@@ -20,6 +20,7 @@
 
 # This makes Rspec fail if there's any output
 RSpec::Matchers.define_negated_matcher :avoid_outputting, :output
+RSpec::Matchers.define_negated_matcher :exclude, :include
```

Comment:
🎉 I noticed this matcher was missing on one of my own PRs and I forgot how to add negated matchers so I just wrote `.not_to include` and forgot to look into it :joy:

Reply:
Nice addition!!